namui-cli/src/services/bundle/lexicon/lexer.rs

```diff
@@ -254,10 +267,10 @@ mod test {
             ],
         };
 
-        let expected_namui_bundle_manifest = NamuiBundleManifest::new(
-            vec![IncludeOperation::new(include_src_path, include_dest_path)],
-            vec![ExcludeOperation::new(exclude_path)],
-        );
-        assert_eq!(namui_bundle_manifest, expected_namui_bundle_manifest);
+        let expected = ParseResult {
+            include: vec![IncludeOperation::new(include_src_path, include_dest_path)],
+            exclude: vec![ExcludeOperation::new(exclude_path)],
+        };
+        assert_eq!(expected, result);
```

Comment:
Previously lexer knew upper object 'manifest' so I abstracted it into ParseResult.